shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -187,14 +182,56 @@ public Intent build(@NonNull Context context) {
 
   @Override
   protected void onCreate(@Nullable Bundle savedInstanceState) {
+    switchLaunchThemeForNormalTheme();
+
     super.onCreate(savedInstanceState);
+
     configureWindowForTransparency();
     setContentView(createFragmentContainer());
-    showCoverView();
     configureStatusBarForFullscreenFlutterExperience();
     ensureFlutterFragmentCreated();
   }
 
+  /**
+   * Switches themes for this {@code Activity} from the theme used to launch this
+   * {@code Activity} to a "normal theme" that is intended for regular {@code Activity}
+   * operation.
+   * <p>
+   * This behavior is offered so that a "launch screen" can be displayed while the
+   * application initially loads. To utilize this behavior in an app, do the following:
+   * <ol>
+   *   <li>Create 2 different themes in style.xml: one theme for the launch screen and
+   *   one theme for normal display.
+   *   <li>In the launch screen theme, set the "windowBackground" property to a {@code Drawable}
+   *   of your choice.
+   *   <li>In the normal theme, customize however you'd like.
+   *   <li>In the AndroidManifest.xml, set the theme of your {@code FlutterActivity} to
+   *   your launch theme.
+   *   <li>Add a {@code <meta-data>} property to your {@code FlutterActivity} with a name
+   *   of "io.flutter.embedding.android.NormalTheme" and set the resource to your normal
+   *   theme, e.g., {@code android:resource="@style/MyNormalTheme}.
+   * </ol>
+   * With the above settings, your launch theme will be used when loading the app, and
+   * then the theme will be switched to your normal theme once the app has initialized.
+   */
+  private void switchLaunchThemeForNormalTheme() {
+    try {
+      ActivityInfo activityInfo = getPackageManager().getActivityInfo(getComponentName(), PackageManager.GET_META_DATA);
+      int normalThemeRID = activityInfo.metaData.getInt("io.flutter.embedding.android.NormalTheme", -1);
+      if (normalThemeRID != -1) {
+        setTheme(normalThemeRID);
+      }
+    } catch (PackageManager.NameNotFoundException exception) {
+      Log.e(TAG, "Could not read meta-data for FlutterActivity.");
```

Comment:
Could not read meta-data for FlutterActivity. Leaving the launch theme in place.